### ISSUE TITLE
[BUG] Continuity Emissivity Breaks Alongside CTM Glowing

### ISSUE THREAD

**Issue by CraftyZombie** [PepperCode1/Continuity#573]
Just wanna know if I should collect more information and file a proper bug report.
Have a pack that adds emissivity to Malum's Brilliant Stone, but when I have CTM installed, Continuity's system doesn't appear to trigger.
This is what shows up with CTM:
<img width="486" alt="Screen Shot 2024-12-23 at 3 04 47 PM" src="https://github.com/user-attachments/assets/1634572e-451f-44c3-9031-66d37a0d0087" />

This is what shows up without CTM:
![2024-12-23_15 06 15](https://github.com/user-attachments/assets/79ec5a2a-327e-46a6-9be3-88a7ad491887)

Is there any way to get them to work at the same time so that the block both gives off light (CTM) but also renders pixels in fullbright (Continuity)?

**Comment by PepperCode1** [comment]
Continuity has no integration with the CTM mod. This is a bug, and I do not know the cause of it.

**Comment by CraftyZombie** [comment]
Only Continuity:
![2024-12-25_17 24 10](https://github.com/user-attachments/assets/49cf7a34-e7e8-493c-bf1b-b4881c8b67ea)
Logs: https://gist.github.com/CraftyZombie/d92ba91709d7d7097e7eeb9e45996771

Both Continuity and CTM:
![2024-12-25_17 26 03](https://github.com/user-attachments/assets/8e8998f1-b12d-4f58-a5fb-8aaa4981d5cf)
Logs: https://gist.github.com/CraftyZombie/e460089fc8802346b55d12fcd81248f1

**Comment by CraftyZombie** [comment]
https://github.com/Chisel-Team/ConnectedTexturesMod/issues/246

**Comment by PepperCode1** [comment]
> ...the block both gives off light (CTM)..

CTM does not do that. Its definition for light only affects the minimum lightmap value of quads which use the corresponding texture, which is similar to what Continuity's emissives do.

One possible reason for why this issue happens is that CTM is wrapping Continuity's model instead of the other way around. This would completely disable connected or emissive textures defined through Continuity for that block/item. If this is the actual cause, then CTM can change that if they wish, but that is more of a Connector/Forgified Fabric API bug than a CTM bug or Continuity bug. However, fixing it would be quite difficult.

**Comment by CraftyZombie** [comment]
> CTM does not do that. Its definition for light only affects the minimum lightmap value of quads which use the corresponding texture, which is similar to what Continuity's emissives do.

My mistake. I was trying to distinguish the effects of the two mods' emissivities.

**Comment by Patomosm** [comment]
I don’t know if the cause of the issue could be the same, but enabling Continuity’s emissivity in the video settings produces this strange visual effect - look at crimson stem in the video below.
However, the issue doesn’t seem to be consistent with the player’s movements. What I mean is that the emissivity disappears apparently at random, without any fixed rule related to distance and/or viewing angle.
In the video it’s Continuity for the newly released Minecraft 26.3 — no shader mode using the FeelTrue texture pack.

https://github.com/user-attachments/assets/16b9341a-9e86-46dd-b1cf-d296d6c4c65e

**Comment by PepperCode1** [comment]
No, that is a completely different issue. Do not use this ticket for it.
